notebooks/226-bert-fill-mask/226-fill-mask-bert-uncased.ipynb

```diff
@@ -0,0 +1,391 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "1007aeb7",
+   "metadata": {},
+   "source": [
+    "### Importing Packages"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fc5168e0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from utils import tokenizer as tk\n",
```

Comment:
do we really need own self written tokenizer or original tokenizer from transformers can be used?

Reply:
I've made my custom tokenizer using the bert based tokenizer. The original tokenizer doesn't add a [MASK] token at the end of sentence if no token is applied. It automatically rejects the input, but I've added the feature where the check happens if there's no [MASK] token, the tokenizer will add the token at the end of the sentence and return the encoded input.